installation.md

```diff
@@ -77,6 +77,11 @@ The most recent version of the tool can be retrieved from the [official release
 page](https://github.com/kubevirt/kubevirt/releases).
 
 ## Deploying on OpenShift
+
+There are three following ways how to deploy KubeVirt on OpenShift
```

Comment:
There exist three ways on how to deploy KubeVirt on OpenShift.

Reply:
Addressed